pytensor/compile/mode.py

```diff
@@ -585,6 +602,8 @@ def get_target_language(mode=None) -> tuple[Literal["py", "c", "numba", "jax"],
         return ("py",)
     if isinstance(linker, CLinker):
         return ("c",)
+    if isinstance(linker, MLXLinker):
+        return ("py",)
```

Comment:
This seems wrong?

Reply:
Can likely remove. What is the purpose of this function? Is it an issue if I remove and it is added if needed in the future?